src/common/dialog/AccessDialog.svelte

```diff
@@ -271,14 +285,20 @@
         </div>
       {/if}
     </div>
+    <div>
+      <input type="checkbox" bind:checked={noindex} />
+      <label class="normal-label">
```

Comment:
Can you put the input inside the label or add a `for` attribute to the label, so that clicking on the label toggles the checkbox?